dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -31,63 +33,132 @@ const Snap100 = ({
 					imagePosition="right"
 					imagePositionOnMobile="left"
 					imageSize="medium"
-					trailText={snap.trailText}
+					trailText={snaps[0].trailText}
 				/>
 			</LI>
 		</UL>
 	);
 };
 
-export const DynamicPackage = ({
-	groupedTrails,
+const Card100 = ({
+	cards,
 	containerPalette,
 	showAge,
-}: Props) => {
-	// Take the first 'snap' - all others are treated as standards
-	const snap = [...groupedTrails.snap].shift();
-	const [primary, ...remaining] = [
-		...groupedTrails.snap.slice(1),
-		...groupedTrails.standard,
-	];
-
-	// TODO: Different layouts are required when there are 4+ 'remaining' cards
-	// dynamic/package will display up to 8 'remaining' cards before moving to 'show more'
-	if (primary.isBoosted) {
-		return (
-			<>
-				{!!snap && (
-					<Snap100
-						snap={snap}
-						containerPalette={containerPalette}
-						showAge={showAge}
-					/>
-				)}
-				<UL>
-					<LI padSides={true}>
+	ginormousHeadline,
+}: {
+	cards: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+	ginormousHeadline?: boolean;
```

Comment:
This kind of thing is so tricky. I guess you could have a `GinormousCard100` or maybe a `isBoosted` prop 🤷 

My vote would be to have a new component like `GinormousCard100` (we might want to shorten the biggness verbiage there 🤣 ) but I don't think there's a right or wrong here. It's true that once you add code it tends to be sticky but I think it's okay to try things and change them later.

Reply:
I like `GinormousCard100` a lot, but if we went for it, where would the logic for picking this sit? Would we create duplicate layouts for each no. of standards, e.g `'9StandardsBoosted' | '9Standards' | '8StandardsBoosted' | '8Standards' | ...`, or would we put the logic in the rendering section, e.g

```tsx
case '8Standards':
return <>
    ....
    {firstSlice[0].isBoosted ? <GinormousCard100 /> : <Card100>
</>
```

Reply:
Oh, does it actually switch here on if it is boosted or not? If so... `BoostedCard100`?

As for where, I think something like `isBoosted ? <BoostedCard100 /> : <Card100>` could work.

🤔 I also think you could have a `isBoosted: boolean` prop on `Card100`. That is maybe a more reasonable balance?

Reply:
Going to try the `isBoosted` props on `Card100`. I realise now that in `Card100` we could also do card.isBoosted check as well, but do we want that to happen implicitly for Card100?

Reply:
> in Card100 we could also do card.isBoosted check as well

That makes sense to me. We check other properties on the card and vary what gets displayed (supportingContent / trailText) so adding this check feels valid